packages/framework-core/src/services/graphql/common.ts

```diff
@@ -1,7 +1,8 @@
 import { GraphQLList, GraphQLScalarType, GraphQLObjectType, GraphQLType } from 'graphql/type/definition'
-import { AnyClass, PropertyMetadata, UserEnvelope, UUID, GraphQLOperation } from '@boostercloud/framework-types'
+import { AnyClass, UserEnvelope, UUID, GraphQLOperation } from '@boostercloud/framework-types'
 import { GraphQLFieldResolver } from 'graphql'
 import { ReadModelPubSub } from '../pub-sub/read-model-pub-sub'
+import { PropertyMetadata } from 'metadata-booster'
```

Comment:
Shouldn't `metadata-booster` be inside the `@boostercloud` group? Why did you choose to add it outside that package group?